python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -1201,6 +1201,12 @@ interface {{ ethernet_interface.name }}
    {{ auth_failure_fallback_mba }}
 {%         endif %}
 {%     endif %}
+{%     if ethernet_interface.traffic_engineering.enabled is arista.avd.defined(true) %}
+   traffic-engineering
+{%         if ethernet_interface.traffic_engineering.administrative_groups is arista.avd.defined %}
+   traffic-engineering administrative-group {{ ethernet_interface.traffic_engineering.administrative_groups | join(",") }}
+{%         endif %}
```

Comment:
```suggestion
{%     if ethernet_interface.traffic_engineering.administrative_groups is arista.avd.defined %}
   traffic-engineering administrative-group {{ ethernet_interface.traffic_engineering.administrative_groups | join(",") }}
{%     endif %}
```
We can configure the administrative-group without just configure the only traffic-engineering in EOS. So we can remove the nested if.

Reply:
You are right, i mistakenly thought adding the admin group command would automatically add the base traffic-engineering command in EOS, but when retesting i found this not to be true.